### PR TITLE
 	Changes symptom timers and values to account for the fact that ssdisease fires once every two seconds, not one

### PR DESCRIPTION
**Pull request by XDTM**
The time between activations in general has been set to about 60% of the previous value.

I've heard reports of healing symptoms being weaker than normal post-rework. This was largely caused by self-respiration healing at half the speed, no longer negating oxygen damage from asphyxiation. I also corrected the other values as the intention was to heal one point of damage per second, not every two.

### CODE REVIEW

File: code/datums/diseases/advance/advance.dm [hunk 1]
```diff
@@ -294,6 +294,8 @@
 		var/list/L = list()
 		for(var/datum/symptom/S in symptoms)
 			L += S.id
+			if(S.neutered)
+				L += "N"
```
Comment (XDTM) [inline code/datums/diseases/advance/advance.dm]: why is this here, the change was already merged

goddamn git and goddamn branches of branches

**Comment by MrStonedOne** [comment]
In general, things like this should ideally be made to work in absolute time if possible, rather then incrementing or decrementing a number every fire, add or subtract the number of deciseconds since last process, or just generate an absolute time value and compare it.
